autotest/test_order3_hf_commond_chat.py

```diff
@@ -0,0 +1,150 @@
+import allure
+import pytest
+import yaml
+from utils.run_client_chat import hfCommondLineTest
+
+
+@pytest.fixture(scope='class', autouse=True)
+def case_config(request):
+    case_path = './autotest/chat_prompt_case.yaml'
+    print(case_path)
+    with open(case_path) as f:
+        case_config = yaml.load(f.read(), Loader=yaml.SafeLoader)
+    return case_config
+
+
+def getList():
```

Comment:
this function is a bit redundant since you have fixture of case_config. Could remove it.